build/ptf

```diff
@@ -0,0 +1,955 @@
+#!/usr/bin/env python
```

Comment:
We install only library and this lib don't contain this file
Ptf readme not contain any information about it
When I saw ptf code it's look like experimental code rather than stable version, but we have to use it even if it is not suite for our code

Reply:
This look strange actually. We need a file which is not provided by a library. I will try to have a look into it

Reply:
i drilled topic and i removed this file